src/helix-ui/styles/core/hx-tabpanel.less

```diff
@@ -0,0 +1,15 @@
+hx-tabpanel {
+  &:extend(hx-reveal all);
```

Comment:
Should there be an import at the top of the file? Double-check; may be working based on an undoc'd feature of LESS or because order of operations when loading.

Reply:
Explicit is better than implicit :)

Reply:
Addressed on new [L1](https://github.com/rackerlabs/helix-ui/pull/115/files#diff-d20a349f29095e73a4cfc57e2c43f88bR1)